src/otx/algo/detection/heads/distance_point_bbox_coder.py

```diff
@@ -84,3 +84,18 @@ def decode(
         if self.clip_border is False:
             max_shape = None
         return distance2bbox(points, pred_bboxes, max_shape)
+
+    def decode_export(
+        self,
+        points: Tensor,
+        pred_bboxes: Tensor,
+        max_shape: tuple[int, ...] | Tensor | tuple[tuple[int, ...], ...] | None = None,
+    ) -> Tensor:
+        """Decode distance prediction to bounding box for export."""
+        assert points.size(0) == pred_bboxes.size(0)  # noqa: S101
```

Comment:
Is there any reason to use assert instead of raise Error?

Reply:
No reason. I got mmdeploy source as it is and ignored ruff error.

Reply:
Could you change all of assert to if... raise? As you know, `assert` code is removed if python is requested to optimize. I think using if... raise is better than assert.

Reply:
Done at https://github.com/openvinotoolkit/training_extensions/pull/3542/commits/d7a4e92ff4ddef6b226fa00543fa24fbc8233cfd.